js/fybsg.js

```diff
@@ -24,6 +24,12 @@ module.exports = class fybsg extends fybse {
             'markets': {
                 'BTC/SGD': { 'id': 'SGD', 'symbol': 'BTC/SGD', 'base': 'BTC', 'quote': 'SGD' },
             },
+            'status': {
+                'status': 'shutdown',
```

Comment:
Yeah! I'll review it once again thoughtfully, and hope to merge it in the coming days.

Reply:
yep sure